packages/ember/src/components/footer.gts

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+
+interface FooterSignature {
+  Element: HTMLElement;
+  Blocks: {
+    left?: [];
+    right?: [];
+    default?: [];
+  };
+}
+
+export default class Footer extends Component<FooterSignature> {
+  get className() {
```

Comment:
Should we refactor this to be `classList` for consistency with the other components?